code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -376,10 +372,12 @@
 
 	size = SIZE_BIG
 
-/mob/living/simple_animal/hostile/asteroid/goliath/OpenFire()
-	visible_message("<span class='warning'>The [src.name] digs its tentacles under [target.name]!</span>")
+/mob/living/simple_animal/hostile/asteroid/goliath/OpenFire(atom/ttarget)
+	if(istype(ttarget))
+		visible_message("<span class='warning'>\The [src] digs its tentacles under \the [ttarget]!</span>")
+
 	playsound(loc, 'sound/weapons/whip.ogg', 50, 1, -1)
-	var/tturf = get_turf(target)
+	var/tturf = get_turf(ttarget)
```

Comment:
I didn't remove anything, I changed goliath's OpenFire() proc to be independent of the goliath's target, and create tentacles under the atom that's passed as an argument. Since hostile mob AI always passes the target as an argument, this change does nothing except allow player goliaths to create tentacles wherever they want.